wiki/Installation/macOS/tr.md

```diff
@@ -0,0 +1,20 @@
+# macOS için kurulum
+
+Bu sayfa size macOS cihazınızda osu!'yu nasıl çalıştıracağınızı kabaca açıklayacaktır.
+
+## Minimum sistem gereksinimleri
+
+- macOS çalıştıran ortalama bir bilgisayar.
+
+## osu!'yu kurmak
+
+osu!'nun macOS sürümü deneyseldir. Oyun işlevsel bir şekilde çalışmayabilir, hatta açılmayabilir. Bu [eğitici video](https://www.youtube.com/watch?v=3ay221bMHNo) sizin aşağıdaki süreci anlamanıza yardımcı olacaktır.
+
+1. Resmi olmayan [indirme sayfasına](https://osu.ppy.sh/community/forums/topics/682197) gidin, `osu!.app.zip` dosyasını indirin ve dizine çıkarın.
+2. Bir uygulama gibi gözüken `osu!.app` klasörünü göreceksiniz. Üzerine sağ tıklayın, ve `Aç`'a tıklayın.
+   - macOS uygulamanın tanımlanamayan bir geliştirici tarafından geliştirilmiş olarak tanıyabileceğinden, dosyaya çift sol tıklamanız onu açmayabilir. Sağ tıklamak ve `Aç`'ı seçmek bu sorunu çözecektir.
+3. Yükleyici otomatik olarak çalışacaktır. Kurulum konumu değiştirilemez ve doğrudan `osu!.app` üzerine kurulur. `osu!.app`'i taşımanız onunla birlikte gelen tüm dosyaları da beraberinde taşıyacaktır. Klasör içerisindeki dosyaları görüntülemek için sağ tıklayıp `Paket İçeriğini Göster` seçeneğini de seçebilirsiniz.
```

Comment:
would be best if we checked the official macOS translation of `Paket İçeriğini Göster`